requirements/base/ogb.txt

```diff
@@ -0,0 +1 @@
+ogb >= 1.0, < 2.0
```

Comment:
Ah damn, didn't realize it has all those dependencies... But I think we cannot rely on ogb_lite since it seems very non-official and not by the stanford team. I guess since `ogb` is an optional dependency it's not so bad